### DISCUSSION TITLE
Moon function

### DISCUSSION THREAD

**Discussion by FileEX** [multitheftauto/mtasa-blue#729] [Ideas]
**Is your feature request related to a problem? Please describe.**
Already we have 'setSunSize' and 'setSunColor' functions and setMoonSize, but we haven't 'setMoonColor' function, it'll would be usefull e.g on 'apocalyptic' gamemode.

**Describe the solution you'd like**
Adding setMoonColor function

**Describe alternatives you've considered**
We can make it with using shader and texture but if we have setSunColor, so why we can't have setMoonColor.

**Additional context**

**Comment by CrosRoad95** [comment]
use shader 🙄 and set custom moon graphic

**Comment by FileEX** [comment]
So, setSunColor is unnecessary, because we can change sun color with shader and custom graphic.

**Comment by Wolfee-J** [comment]
You cannot modify the moon at all at the moment @CrosRoad95

**Comment by Dutchman101** [comment]
> You cannot modify the moon at all at the moment @CrosRoad95

You can, i took the screenshot below after replacing the moon with a planet;

![image](https://user-images.githubusercontent.com/923242/50401588-bd55b180-0787-11e9-8ea5-2a6c0704fc58.png)


Resource: [moonmod.zip](https://github.com/multitheftauto/mtasa-blue/files/2707427/moonmod.zip)

**Comment by Wolfee-J** [comment]
Well that's odd, last time I attempted I could not find anything on it.
